cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -155,7 +170,7 @@ func (r *KubeArchiveConfigReconciler) reconcileA14eServiceAccount(ctx context.Co
 		return sa, err
 	}
 
-	err = r.Get(ctx, types.NamespacedName{Name: a14eName, Namespace: kaconfig.Namespace}, &corev1.ServiceAccount{})
+	err = r.Get(ctx, types.NamespacedName{Name: a14eName, Namespace: k9eNs}, &corev1.ServiceAccount{})
```

Comment:
With this change, `kaconfig` is not used

Reply:
Fixed.